.github/workflows/docker-image.yml

```diff
@@ -32,6 +32,9 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Get current date
+        run: echo "BUILD_DATE=\"$(date -u +'%Y-%m-%dT%H:%M:%SZ')\" >> ${GITHUB_ENV}"
```

Comment:
We usually start environment variables with the prefix `TYPST_` to prevent collisions with other applications.